docs/features/wired-mesh.rst

```diff
@@ -56,32 +56,54 @@ the ``mesh`` role to the ``interfaces.*.default_roles`` options in site.conf.
 Commandline
 ===========
 
+Since version 2022.1, the wired network configuration is rebuilt from ``/etc/config/gluon`` on each ``gluon-reconfigure``, and therefore overwritten at least on every update of the node.
+There are currently three roles available for interfaces: ``client``, ``mesh`` and ``uplink``.
+
+Be careful: the ``client`` role must not be mixed with the other two.
+
+In order to make persistent changes to the routers configuration it's therefore neccessary to:
+
+* change the sections in ``/etc/config/gluon`` (see examples below)
+* call ``gluon-reconfigure`` to regenerate ``/etc/config/network``
+* restart networking using either ``/etc/init.d/network restart`` or cleaner ``reboot`` the device
+
 Enable Mesh-on-WAN::
 
-  uci set network.mesh_wan.disabled=0
-  uci commit network
+  config interface 'iface_wan'
+	option name '/wan'
+	list role 'mesh'
+	list role 'uplink'
 
 Disable Mesh-on-WAN::
 
-  uci set network.mesh_wan.disabled=1
-  uci commit network
+  config interface 'iface_wan'
+	option name '/wan'
+	list role 'uplink'
 
 Enable Mesh-on-LAN::
 
-  uci set network.mesh_lan.disabled=0
-  for ifname in $(cat /lib/gluon/core/sysconfig/lan_ifname); do
-    uci del_list network.client.ifname=$ifname
-  done
-  uci commit network
+  config interface 'iface_lan'
+	option name '/lan'
+	list role 'mesh'
 
 Disable Mesh-on-LAN::
 
-  uci set network.mesh_lan.disabled=1
-  for ifname in $(cat /lib/gluon/core/sysconfig/lan_ifname); do
-    uci add_list network.client.ifname=$ifname
-  done
-  uci commit network
+  config interface 'iface_lan'
+	option name '/lan'
+	list role 'client'
+
+It is furthermore possible to make use of VLANs.
+The following section would create a VLAN with id 8 on ``eth0`` and emit the mesh network on it::
+
+  config interface 'iface_lan_vlan8'
+	option name 'eth0.8'
+	list role 'mesh'
+
+Other VLAN-interfaces could be configured on the same parent interface in order to have all three roles avaliable on ``eth0`` without having them interfere with each other.
+This feature comes in especially handy in the persistent configuration of virtual machines as offloader for bigger installations.
+
```

Comment:
```suggestion

It is possible to configure multiple interfaces with the same role, but currently all those interfaces will be bridged without any isolation. 
Mesh nodes that are connected to one of the mesh interfaces of a node can see each other directly.

```

sadly it doesn't look like #2484 (or something like it) will be merged in time for 2022.1

but idk, maybe it's better to leave this "disclaimer" away

Reply:
I agree on the other two points, but looking at the fact that we intend to change the behaviour, I'd avoid explicitly documenting that there's no isolation.